.taskcluster.yml

```diff
@@ -7,6 +7,8 @@ metadata:
 tasks:
   - provisionerId: "{{ taskcluster.docker.provisionerId }}"
     workerType: "{{ taskcluster.docker.workerType }}"
+    scopes:
+        - "auth:aws-s3:read-only:taskcluster-raw-docs/*"
```

Comment:
That may be the case with the SDK, but for AWS in general, if you set a resource to be readable by "everyone", then it can be read with a regular old GET request to the API endpoint.